omnibus-sepolia.toml

```diff
@@ -452,3 +458,38 @@ args = [
     "<%= settings.weth_address %>",
     "<%= settings.weth_rewards_distributor %>",
 ]
+
+#  adding these for future reference, it should be ensured these are registered as well
```

Comment:
What about these calls? Why dont we enable them now?

Reply:
They've been manually executed on the sepolia testnet so cannon build breaks if I uncomment them here, I'll add these properly in the mainnet toml

Reply:
Please talk to DB and use cannon `alter` to add these into the state without breaking. These are not the "transient" steps that should be executed without config changes. It is not a blocker for this PR though but please do follow up

Reply:
Thanks, I didn't know about the cannon alter command.

Reply:
Given we now have the test to validate rewards distributors registrations - #426, current sepolia deployment will fail it and we definitely need to add these invokes to configs. Can be done in a separate PR (and the test from #426 must be added for BFP too - with its own flavour)

Reply:
I think it makes sense for this to go into a separate PR